controllers/storagecluster/exporter.go

```diff
@@ -709,3 +715,86 @@ func createMetricsExporterRolebindings(ctx context.Context, r *StorageClusterRec
 
 	return nil
 }
+
+func createRookCephClusterRolebindings(ctx context.Context,
+	r *StorageClusterReconciler, instance *ocsv1.StorageCluster) error {
+	// rookCephMonitorMgrRoleBinding is a cluster rolebinding for monitor mgr
+	rookCephMonitorMgrRoleBinding := &rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rook-ceph-monitor-mgr",
+			Namespace: instance.Namespace,
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     "rook-ceph-monitor-mgr",
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind: "ServiceAccount",
+				Name: "rook-ceph-mgr",
+			},
+		},
+	}
+
+	// rookCephMonitorRoleBinding is a cluster rolebinding for rook-ceph monitor
+	rookCephMonitorRoleBinding := &rbacv1.ClusterRoleBinding{
```

Comment:
```suggestion
	rookCephMonitorRoleBinding := rbacv1.ClusterRoleBinding{
```
Since we're just copying these values later, do we need this to be a pointer?

Reply:
Nope.